### ISSUE TITLE
Scrolling in Veripaq Analzyer Tables is unbearable slow

### ISSUE THREAD

**Issue by fmms** [DaxStudio/DaxStudio#220]
Hi,

thanks alot for integrating Vertipaq analyer in Dax Studio.

I found an issue with the current version in my environment. Scrolling in the table below takes about 1 second per row:

![image](https://user-images.githubusercontent.com/908734/65376495-bc16ce00-dca0-11e9-8841-12485e58dc47.png)

This is a remote desktop session. However, with no other application I am experiencing such issues.

Nor does it happen in other tables in Dax Studio, e.g. 
![image](https://user-images.githubusercontent.com/908734/65376632-6e02ca00-dca2-11e9-8b84-2dde730672a9.png)

Thanks alot

**Comment by marcosqlbi** [comment]
How many tables/columns do you have in the model? Can you share the VPAX file?

**Comment by fmms** [comment]
I Marco I have sent you an email.

The model does not seem to be that bad. It has in total less than 300 columns. The main table has about 160 columns.

regards

**Comment by dgosbell** [comment]
What is the scrolling like on the "Columns" tab in the Vertipaq Analyzer view? This issue could be some combination of RDP and the grouping in the Tables tab.

**Comment by fmms** [comment]
That seems to be the correct analysis. "Columns" tab works fine.

**Comment by dgosbell** [comment]
This could be tricky to pin down. I just tested by using Remote Desktop to a dev server and Analyzing a model with 900 columns and the scrolling was pretty smooth. There were a few minor pauses, but nothing like 1 second per row

**Comment by fmms** [comment]
I have just tried this on my local machine and there I do not have the scrolling issues. So it seems to be a problem that only occurs with remote desktop connections. However, I do not have this problem with other Apps on the same machine. It is an Azure VM.

**Comment by dgosbell** [comment]
Yes, I can't reproduce this issue on my local machine or using RDP against one of the dev servers at work. I'll try and test against an Azure VM.

**Comment by dgosbell** [comment]
Actually I've been working on this feature as part of the next release. If your interested you can try a preview build of the portable zip release from our build server here [https://ci.appveyor.com/project/DarrenGosbell/daxstudio/builds/28030852/artifacts](https://ci.appveyor.com/project/DarrenGosbell/daxstudio/builds/28030852/artifacts)

**Comment by fmms** [comment]
Hi,

I installed the latest build:
![image](https://user-images.githubusercontent.com/908734/66717157-28837980-edd6-11e9-928d-f9da408f31b8.png)

However, I cannot recognize any difference regarding performance on my Azure VM.

**Comment by dgosbell** [comment]
@fmms - yes, because I can't currently reproduce this issue I have not changed anything yet. 

I'm not running any code on a per row basis, so this is probably a bug/issue with the Microsoft data binding framework. I'll try to find some time this week to spin up a test Azure VM. If I can reproduce the issue I can try changing some of the binding options to see if I can find some combination of options that works in both scenarios.

**Comment by dgosbell** [comment]
I still can't reproduce this issue. I created a test VM in an Azure data centre on the other side of the world from me and the scrolling of the Vertipaq Analyzer view is no slower for me than any of the other grids.

Are you using a direct RDP connection or are you using a Bastion connection?

**Comment by fmms** [comment]
I am using a standard RDP connection with the native Win 10 RDP client.

**Comment by dgosbell** [comment]
hmm, that's exactly what I did too and it worked fine for me.
